manifests/bosh-manifest/scripts/generate-bosh-certs.sh

```diff
@@ -1,89 +0,0 @@
-#!/bin/sh
-
-set -e
-set -u
-
-CA_NAME="bosh-CA"
-
-# List of certs to generate
-# Format:
-#
-# <name_cert>,<domain1>[,domain2,domain3,...]
-#
-# Note: ALWAYS add a comma after <name_cert>, even if there are no domains
-#
-CERTS_TO_GENERATE="
-default.nats.bosh-internal,bosh.${SYSTEM_DNS_ZONE_NAME}
-default.director.bosh-internal,
-default.hm.bosh-internal,
-bosh_director,bosh.${SYSTEM_DNS_ZONE_NAME},bosh-external.${SYSTEM_DNS_ZONE_NAME}
```

Comment:
These SAN's aren't reflected in the cert generated in the manifest. This means that for a clean deployment things fail with cert errors eg:
```
  Performing request GET 'https://bosh.alext.dev.cloudpipeline.digital:25555/info':
    Performing GET request:
      Retry: Get https://bosh.alext.dev.cloudpipeline.digital:25555/info: x509: certificate is not valid for any names, but wanted to match bosh.alext.dev.cloudpipeline.digital
```